src/components/_global/BalInlineInput/BalInlineInput.vue

```diff
@@ -98,12 +98,18 @@ const inputAttrs = computed(() => omit(attrs, 'class'));
  */
 function toggleEditable() {
   isEditable.value = !isEditable.value;
+  emit('editToggled', isEditable.value);
   setTimeout(() => {
     if (inputElement.value) {
       inputElement.value.focus();
     }
   }, 200);
 }
+
+function handleBlur(e: HtmlInputEvent) {
```

Comment:
on blur of the component, toggling it to not editable seems to be the good UX move